### PR TITLE
Updated the installation guide

### PR DESCRIPTION
**Pull request by mcinerneym**
Modified the Installation Guide to be more accurate and updated the pictures

### CODE REVIEW

**Comment by evanho9** [comment]
Looks good! Tested by team Manchester

**Comment by nguyenvic** [comment]
Looks good and thorough!

**Review by evanho9 — APPROVED** [review]

**Review by nguyenvic — APPROVED** [review]

File: Installation_Guide.md [hunk 1]
```diff
@@ -169,67 +180,96 @@ pip install python-dotenv
...
+		
 			[Unit]
 			Description=Raspberry Pi Monitoring Manager
 			After=network.target
 
 			[Service]
 			Type=simple
-			WorkingDirectory=/usr/bin/piman/monitoring
-			ExecStart=/usr/bin/piman/monitoring/monitoring_piman.sh
+			WorkingDirectory=/home/usr/piman/monitoring
```
Comment (breed) [inline Installation_Guide.md]: need to discuss this to agree on the path between PRs

File: Installation_Guide.md [hunk 2]
```diff
@@ -169,67 +180,96 @@ pip install python-dotenv
...
+			ExecStart=sh /home/usr/piman/monitoring/monitoring_piman.sh
```
Comment (breed) [inline Installation_Guide.md]: you shouldn't need the "sh "

**Review by breed — CHANGES_REQUESTED** [review]
fix the paths and you should be good to go.

File: Installation_Guide.md [hunk 3]
```diff
@@ -190,18 +190,18 @@ For example:
 
 			[Service]
 			Type=simple
-			WorkingDirectory=/home/usr/piman/monitoring
-			ExecStart=sh /home/usr/piman/monitoring/monitoring_piman.sh
+			WorkingDirectory=/home/usr/local/share/piman/monitoring
```
Comment (breed) [inline Installation_Guide.md]: these paths are not correct.

**Review by breed — CHANGES_REQUESTED** [review]

**Review by breed — APPROVED** [review]
thank you

**Pull request merged**